modules/archiving-a-pipeline-run.adoc

```diff
@@ -0,0 +1,38 @@
+:_module-type: PROCEDURE
+
+[id="archiving-a-pipeline-run_{context}"]
+= Archiving a pipeline run
+
+[role='_abstract']
+You can retain records of your pipeline runs by archiving them. If required, you can restore runs from your archive to reuse, or delete runs that are no longer required.
```

Comment:
question: do you have to restore a run before you can delete it? I'm guessing not. 

No. Looking at the UI in the latest build, you can only delete a run from the archive. You can't delete a run from the active runs tab, from what I can see.

Is further explanation needed? To clean up the list of runs so that only the ones that you are currently using are visible??

Possibly, yes. 


.

Reply:
This was discussed in a slack chat recently, but the conclusions were unclear. I am not sure if this is a bug or if it is intended behavior. I will try and clarify this on Monday. My understanding is that a user should be able to delete an active run, and this feature should be available in the UI. I don't see a need to force a user to archive a run that they don't really want to archive. I'll clarify this and get back to you.

Reply:
I'll exclude the  "Deleting an active pipeline run" module from the assembly, as I see no evidence that this functionality is available.